rosa_tutorials/rosa-mobb-verify-permissions-sts-deployment.adoc

```diff
@@ -0,0 +1,117 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-verify-permissions-sts-deployment"]
+= Tutorial: Verifying Permissions for a ROSA STS Deployment
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-verify-permissions-sts-deployment
+
+toc::[]
+
+// ---
+// date: '2022-10-04'
+// title: "Verify Permissions for ROSA STS Deployment"
+// authors:
+//   - Tyler Stacey
+//   - Kumudu Herath
+// tags: ["AWS", "ROSA", "STS"]
+// ---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+To proceed with the deployment of a ROSA cluster, an account must support the required roles and permissions.
+AWS Service Control Policies (SCPs) cannot block the API calls made by the installer or operator roles.
+
+Details about the IAM resources required for an STS-enabled installation of ROSA can be found here: xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS]
+
+This guide is validated for ROSA v4.11.X.
+
+== Prerequisites
+
+* link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[AWS CLI]
+* xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-get-started-cli[ROSA CLI] v1.2.6
+* link:https://stedolan.github.io/jq/[jq CLI]
+* link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_testing-policies.html[AWS role with required permissions]
+
+[id="verify-ROSA-permissions_{context}"]
+== Verifying ROSA permissions
+To verify the permissions required for ROSA we can run the script below without ever creating any AWS resources.
+
+The script uses the `rosa`, `aws`, and `jq` CLI commands to create files in the working directory that will be used to verify permissions in the account connected to the current AWS configuration.
+
+The AWS Policy Simulator is used to verify the permissions of each role policy against the API calls extracted by `jq`; results are then stored in a text file appended with `.results`.
+
+This script will verify the permissions for the current account and region:
```

Comment:
See below comment, but if we remove the script, we could replace the text here:

> To verify the permissions required for ROSA we can run the script included in the below section without ever creating any AWS resources.
> 
> The script uses the rosa, aws, and jq CLI commands to create files in the working directory that will be used to verify permissions in the account connected to the current AWS configuration.
> 
> The AWS Policy Simulator is used to verify the permissions of each role policy against the API calls extracted by jq; results are then stored in a text file appended with .results.
> 
> This script is designed to verify the permissions for the current account and region.

Reply:
Okay so if I'm interpreting this correctly, I may as well delete the entirety of the following script (lines 46-77) and then replace this entire "Verifying ROSA permissions" section with the suggested text in this comment. Let me know if that's not correct.